### PR TITLE
add some auth0 boilerplate, in progress

### PR DESCRIPTION
**Pull request by joyhalliday**

### CODE REVIEW

**Review by dorntrevor7 — APPROVED** [review]
good to go

File: client/src/pages/profile.js [hunk 1]
```diff
@@ -1 +1,23 @@
-//current user and other users
\ No newline at end of file
+//current user and other users
```
Comment (dorntrevor7) [inline client/src/pages/profile.js:1]: good

**Review by dorntrevor7 — DISMISSED** [review]
good to go

**Review by dorntrevor7 — APPROVED** [review]

**Review by skylardorn — APPROVED** [review]
Nice

**Pull request merged**